### PR TITLE
Implement case insensitive answers

### PR DESCRIPTION
**Pull request by nikitabelonogov**

### CODE REVIEW

File: test/test_dialog_engine.py [hunk 1]
```diff
@@ -52,6 +52,49 @@ def test_some_answers(self):
             print(k, v, answer)
             self.assertTrue(answer in v)
 
+    def test_some_case_insensitive(self):
+        ls = {
+            'Привет': [
+                'ну привет',
+                'тебя не звали, иди отсюда'
+            ],
+            'ПрBвЕт': [
```
Comment (wlad031) [inline test/test_dialog_engine.py]: mistyping?

**Review by wlad031 — CHANGES_REQUESTED** [review]

File: test/test_dialog_engine.py (hunk 1 above)
Comment (nikitabelonogov) [reply test/test_dialog_engine.py]: no

Comment (nikitabelonogov) [reply test/test_dialog_engine.py]: yes

**Review by wlad031 — DISMISSED** [review]

**Review by wlad031 — APPROVED** [review]

**Comment by wlad031** [comment]
Could you release this version?

**Comment by nikitabelonogov** [comment]
Ok, I've also ban opportunity to perform merge and rebase merging in PRs just FYI

**Pull request merged**